erpnext/accounts/general_ledger.py

```diff
@@ -53,25 +53,26 @@ def merge_similar_entries(gl_map):
 		# if there is already an entry in this account then just add it
 		# to that entry
 		same_head = check_if_in_list(entry, merged_gl_map)
+
 		if same_head:
-			same_head.debit	= flt(same_head.debit) + flt(entry.debit)
-			same_head.debit_in_account_currency	= \
-				flt(same_head.debit_in_account_currency) + flt(entry.debit_in_account_currency)
-			same_head.credit = flt(same_head.credit) + flt(entry.credit)
-			same_head.credit_in_account_currency = \
-				flt(same_head.credit_in_account_currency) + flt(entry.credit_in_account_currency)
+			same_head["debit"] = flt(same_head["debit"]) + flt(entry["debit"])
```

Comment:
This changes is not required if you use frappe._dict() while making gl_entries dict.